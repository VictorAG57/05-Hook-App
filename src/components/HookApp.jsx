import React from 'react';

const HookApp = () => {
    return (
        <div>
            <h1>Hola Mundo - Test</h1>
        </div>
    )
}

export default HookApp
