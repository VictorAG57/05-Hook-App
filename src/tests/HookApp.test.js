import React from "react";
import { shallow } from "enzyme";
import '@testing-library/jest-dom';
import HookApp from "../components/HookApp";

describe('Pruebas en el componente HookApp', ()=> {
    test('CUsar un snapShot ', () => {

        const wraper = shallow(<HookApp></HookApp>);

        expect( wraper ).toMatchSnapshot();
    });
}); 