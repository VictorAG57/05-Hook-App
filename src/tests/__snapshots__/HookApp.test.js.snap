// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente HookApp CUsar un snapShot  1`] = `
<div>
  <h1>
    Hola Mundo - Test
  </h1>
</div>
`;
